docs/quickstart.md

```diff
@@ -0,0 +1,141 @@
+# quickstart
+As described in the [README](https://github.com/neume-network/strategies/readme.md), a strategy requires an `extractor` and, optionally, a `transformer`:
+
+- every run of the `update` function of the `extractor` should pull a chunk of the data from the source, be it a page, a time range etc
+- once the extraction process is finished, the transformation part begins. This is executed on a per-line basis (`onLine`) and should prepare the data to be compliant with the schema of the final dataset.
+
+Depending on the data source, different initial configurations may be needed.
+
+## https strategy
+A web 2 strategy crawls data from an `https` source (rss feed, apis, etc...). 
+
+This guide will show you the steps to crawl XKCD.
+
+First off, the `core` expects, both for `init` and `update`, a message compliant to the [schema definition for https strategies](https://github.com/neume-network/schema/blob/main/src/schema.mjs).
+
+the `init` function message should provide the starting state of the crawling. For XKCD, we would write something like:
+```
+{
+    write: null,
+    messages: [
+      {
+        type: "https",
+        commissioner: name,
+        version,
+        options: {
+          url: https://xkcd.com/1/info.0.json,
+          method: "GET",
+          header: null,
+          body: null
+        },
+        results: null,
+        error: null,
+      },
+    ],
+  };
+```
+
+* `write`: specifies the data to be written in string format. `null` during init, as there is no data
+* `type`: the protocol. must be `https` for this type of strategy
+* `commissioner`: ??
+* `version`: can be anything, it's currently not used
+* `options`:
+  * `url`: the URL pointing to the first chunk to crawl. First page on XKCD is `1`
+  * `method`: GET, POST
+  * `headers`: if any specific header should be sent to the source, it shall be specified here (for instance authorization headers)
+  * `body`: same
+* `results`: this field will be used by the core to pass the output of the crawling to the `update` function
+
+Once the `init` message has been returned, the `core` will fetch the first chunk of data and pass it to `update`. 
+
+The function, ideally, should take care of the following:
```

Comment:
Isn't this a bit of a repetition of what is written right below @il3ven ? Or was your intention to have a more summarized description of the `update` role?

Reply:
My main intention was to point out that update is similar to init. Instead of storing, it can also send new messages to worker. For an example you can see [soundxyz-metadata](https://github.com/neume-network/strategies/blob/main/src/strategies/soundxyz-metadata/extractor.mjs#L106).

Reply:
@il3ven i added what you requested, just a bit more consistent with the rest of the sentence. Let me know if it's ok